build-tests/sparo-output-test/etc/checkout-help.txt

```diff
@@ -21,3 +21,5 @@ Options:
                      already exists, reset it to <start-point>         [boolean]
       --profile                                            [array] [default: []]
       --add-profile                                        [array] [default: []]
+      --to                                                 [array] [default: []]
+      --from                                               [array] [default: []]
```

Comment:
How do `--to` and `--from` interact with `--profile`?  Is combined with the profile's selection, to produce the union?

Reply:
updated